app/components/progress_tab/yearly_report_component_v2.rb

```diff
@@ -30,7 +30,11 @@ def render?
     Flipper.enabled?(:yearly_reports_start_from_april, @current_user)
   end
 
-  def last_n_years
+  def is_data_available_for(year)
```

Comment:
Consider avoiding `is`, and using something like `def data_available?(year:)`

Reply:
yeah, will do